colabs/raytune/W&B + RayTune.ipynb

```diff
@@ -6,7 +6,8 @@
       "name": "W&B-RayTune.ipynb",
```

Comment:
<p>This text is good, but needs a little massaging. Here's my suggested edit:</p><p> </p><blockquote>We're now almost ready to call <code>tune.run</code>  to launch our hyperparameter sweep!</blockquote><blockquote>We just need to do two things:</blockquote><blockquote>1.  Configure the <code>wandb</code>  run with things like a project name</blockquote><blockquote>2.  Pass the <code>WandbLogger</code>  to <code>tune.run</code> so we can capture the output of the <code>tune.report</code> in our training loop.</blockquote><blockquote><br></blockquote><blockquote>The second is easy: we just need to import the <code>WandbLogger</code>  and pass it as one of the <code>loggers</code>  to <code>tune.run</code>.</blockquote><blockquote><br></blockquote><blockquote>You can configure the <code>wandb</code> run inside the <code>config</code> dictionary for your <code>tune</code> run. The W&amp;B configuration info should be stored as a dictionary inside <code>config</code> with the key <code>"wandb"</code>.</blockquote><blockquote><br></blockquote><blockquote>The keys and values in that dictionary are then provided as keyword arguments to <code>wandb.init</code> (see <a href="see [the documentation](https://docs.wandb.com/library/init" target="_blank">the documentation</a> for details), with a few exceptions, described in <a href="https://docs.wandb.com/library/integrations/ray-tune#parameters" target="_blank">our documentation of the Ray/Tune integration</a>.</blockquote><blockquote><br></blockquote><blockquote>The most important of these are <code>api_key</code> and <code>api_key_file</code> , one of which must be provided to allow <code>tune</code> to log information on your behalf to W&amp;B. Here, we're entering it manually using the Python library <code>getpass</code>.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>